### DISCUSSION TITLE
2FA UX Improvements

### DISCUSSION THREAD

**Discussion by spencerhooks** [diced/zipline#306] [Ideas]
When prompting for 2FA on login, please automatically make the input field active. This way the user can simply type in their code without having to select the field with the mouse. Also allow for the enter key to proceed with login instead of requiring the user to click the button with the mouse (after entering auth code). 

I always feel like I'm racing the clock with 2FA and it's a better UX when I can just look at my phone and type the code without having to look up and stuff. This is obviously low-priority polish, but it annoys me every time 😁

**Comment by TacticalTechJay** [comment]
I gotcha. #309
